ironfish/src/wallet/account/account.test.ts

```diff
@@ -2179,6 +2179,171 @@ describe('Accounts', () => {
     })
   })
 
+  describe('getUnspentNotesSortedByValue', () => {
+    it('loads notes sorted by value', async () => {
+      const { node } = nodeTest
+      const account = await useAccountFixture(node.wallet)
+
+      const minerBlockA = await useMinerBlockFixture(
+        node.chain,
+        undefined,
+        account,
+        node.wallet,
+      )
+
+      await node.chain.addBlock(minerBlockA)
+      await node.wallet.updateHead()
+
+      const minerBlockB = await useMinerBlockFixture(
+        node.chain,
+        undefined,
+        account,
+        node.wallet,
+      )
+
+      await node.chain.addBlock(minerBlockB)
+      await node.wallet.updateHead()
+
+      const block = await useMinerBlockFixture(node.chain, undefined, account, node.wallet, [
+        await useTxFixture(node.wallet, account, account),
+        await useTxFixture(node.wallet, account, account),
```

Comment:
nit - might be helpful to set the value of these transactions so that there are multiple notes with distinct values